### PR TITLE
Fix licence issues.

### PR DESCRIPTION
**Pull request by pan-**
- provide the file `apache-2.0.txt` as mentionned in `LICENCE`.
- add a apache 2 licence header to script/pick_nrf51_files.py.
- add a apache 2 licence header to script/replace_headers.py.

### CODE REVIEW

**Comment by ciarmcom** [comment]
Automatic CI verification build not done, please verify manually.

**Comment by pan-** [comment]
@rgrover Could you review this ?

File: script/pick_nrf51_files.py [hunk 1]
```diff
@@ -1,5 +1,22 @@
 #!/usr/bin/env python
 
+"""
+mbed SDK
+Copyright (c) 2015-2016 ARM Limited
+
```
Comment (rgrover) [inline script/pick_nrf51_files.py]: could you please add here: `SPDX-License-Identifier: Apache-2.0`?
refer to https://github.com/ARMmbed/mbed-drivers/blob/master/mbed-drivers/AnalogIn.h

File: script/replace_headers.py [hunk 2]
```diff
@@ -1,3 +1,20 @@
+"""
+mbed SDK
+Copyright (c) 2015-2016 ARM Limited
+
```
Comment (rgrover) [inline script/replace_headers.py]: same as above

**Comment by rgrover** [comment]
otherwise looks good. go ahead.

**Comment by pan-** [comment]
I have added the `SPDX-License-Identifier: Apache-2.0` line.
I merge this.

**Pull request merged**